core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/LimitedLineConsumerWrapper.java

```diff
@@ -13,12 +13,13 @@
 import java.io.IOException;
 
 /**
- * Wrapper of {@link LineConsumer} object which is limiting total number of characters which this object can consume.
+ * Wrapper of {@link LineConsumer} object which is limiting total number of characters which this
+ * object can consume.
  *
  * @author Dmytro Nochevnov
  * @author Anatolii Bazko
  */
-public class LimitedSizeLineConsumerWrapper implements LineConsumer {
+public class LimitedLineConsumerWrapper implements LineConsumer {
   private final int limit;
```

Comment:
We are walking by the circle. Can you guarantee that this class limiting number of lines?